submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/FileListSubmissionTest.kt

```diff
@@ -179,34 +181,86 @@ internal class FileListSubmissionTest(private val tempFolder: TemporaryFolder) :
             val createdSub = submissionRepository.getExtByAccNo(accNo)
             val subFolder = "$submissionPath/${createdSub.relPath}"
 
-            val fileListName = createdSub.section.fileList?.fileName
-            val expectedTabFiles = if (mongoMode) {
-                if (enableFire) listOf() else listOf(
-                    NfsFile("$fileListName.json", File(subFolder).resolve("Files/$fileListName.json")),
-                    NfsFile("$fileListName.xml", File(subFolder).resolve("Files/$fileListName.xml")),
-                    NfsFile("$fileListName.pagetab.tsv", File(subFolder).resolve("Files/$fileListName.pagetab.tsv"))
-                )
-            } else listOf()
-
-            val submissionPageTabFiles = if (mongoMode) {
-                if (enableFire) listOf() else listOf(
-                    NfsFile("$accNo.json", File(subFolder).resolve("$accNo.json")),
-                    NfsFile("$accNo.xml", File(subFolder).resolve("$accNo.xml")),
-                    NfsFile("$accNo.pagetab.tsv", File(subFolder).resolve("$accNo.pagetab.tsv"))
-                )
-            } else listOf()
-
-            assertThat(createdSub.section.fileList).isEqualTo(ExtFileList("FileList", pageTabFiles = expectedTabFiles))
-            assertThat((createdSub.pageTabFiles)).isEqualTo(submissionPageTabFiles)
+            if (mongoMode)
```

Comment:
Why are the brackets ( {} ) missing on this `if` statement? This ain't no python kid!

In any case, this shouldn't be necessary. Why should the files be affected by the database if we're consuming extended? The extended submission object should be the same regardless if it was persisted on mongo or mysql

Reply:
Both submission and file list page tab files are not stored in mysql just in mongo.  We did not update schema to support this as is not worth it given we are migrating now.